code/modules/reagents/reagent_containers/spray.dm

```diff
@@ -131,6 +131,9 @@
 	volume = 50
 	list_reagents = list("cleaner" = 50)
 
+/obj/item/reagent_containers/spray/cleaner/drone/cyborg_recharge(coeff, emagged)
```

Comment:
What about the janiborg lube?

Reply:
It's declared locally over in `robot_module.dm`:

![image](https://user-images.githubusercontent.com/42044220/111093879-f2d14b80-8507-11eb-8220-29696b2a97fe.png)